src/Sylius/Component/Shipping/CHANGELOG.md

```diff
@@ -4,3 +4,8 @@ CHANGELOG
 ### v0.10.0
 
 * Initial dev release.
+
+### 19-11-2015
+
+* Remove `Calculator`
+* All calculators class implements `CalculatorInterface`
```

Comment:
I know we miss previous versions, but let's add this to v0.16.0 section. Kudos for remembering about changelog. ;)